ci/jenkins/jobs/projects.yaml

```diff
@@ -325,3 +325,33 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-eks':
+          test_name: conformance-net-policy
+          node: antrea-cloud
+          description: 'This is EKS conformance and network policy test for antrea.'
+          builders:
+          - shell: |-
+              #!/bin/bash
```

Comment:
shebang probably not needed here since you are just invoking a script

Reply:
@edwardbadboy suggested it might be necessary for adding this in jenkins shell build steps

Reply:
`#!/bin/bash` is used when the shell script contains bashism. If it's just invoking other script we can delete `#!/bin/bash`.